lib/user_interface.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class UserInterface
+  CLEAR_COMMAND = "\033[H\033[2J"
+  ERROR_MESSAGE = 'Wrong input. Please try again: '
+  MENU_MESSAGE = "\n---------------------\n\n\s\s\s\sCONTACT BOOK\n\n---------------------\n
+  \n1) Exit the program\n\nChoose a menu option: "
+
+  attr_reader :user_input
+
+  def initialize(input, output)
+    @input = input
+    @output = output
+  end
+
+  def run
+    output.print CLEAR_COMMAND, MENU_MESSAGE
+
+    loop do
+      @user_input = input.gets.chomp
```

Comment:
You don't need this variable to be an instance variable, it can be a normal variable.
This is because of how Ruby works: the last thing at the end of a method will be returned.

So in this case the method would return `user_input`.

then you can call this method like this:

```ruby
user_input = ui.run
```

and in tests you can assert on the value returned by `ui.run`